greenboot.spec

```diff
@@ -21,7 +23,7 @@ Requires:           systemd
 %package motd
 Summary:            Message of the Day updater for greenboot
 Requires:           %{name} = %{version}-%{release}
-Requires:           pam >= 1.3.1
+Requires:           pam >= 1.3.1-15
```

Comment:
+1 on adding the `-15` and adding a selinux-policy Requires once https://github.com/fedora-selinux/selinux-policy/pull/244 is in.

I think there will also be a Requires on `setup >= 2.12.7` once this gets merged https://pagure.io/setup/pull-request/16# - to make sure the `/run/motd` and `/run/motd.d` paths are already created so that the proper context can be applied.